library/general/test/report_test.rb

```diff
@@ -44,15 +44,15 @@
       let(:show) { false }
 
       it "does not show a popup" do
-        expect(Yast::Popup).to_not receive(meth)
+        expect(Yast::Popup).to_not receive("#{meth}Geometry")
```

Comment:
hmm, I though that receive should get symbol and not string. Do you test if it really fail if it is called?

Reply:
well I'm not the author of tests, however it was failing when I patched sources before updating the testsuite ;-)

Reply:
ok, so looks like it accepts also string. So I also learn something new here :+1: